src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,101 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static int DEFAULT_CAPACITY = 16;
+    private static double LOAD_FACTOR = 0.75;
+    private int size = 0;
```

Comment:
redundant value declaration.
```suggestion
    private int size;
```